src/components/Calculator.jsx

```diff
@@ -18,7 +19,8 @@ const Calculator = () => {
 
   return (
     <>
-      <section className="calculator">
+      <section className="calculator flex-spaced">
+        <Quotes />
```

Comment:
- [ ] Instead of here, the `Quote` component should be imported into the main component (`App` component). Therefore you will have something that closely resembles this 👇 
```javascript
const App = () => (
  <>
    <Calculator />
    <Quotes />
  </>
);
```